comma/hypothesis.py

```diff
@@ -26,63 +30,312 @@ class Hypothesis:
-        'medium',
-        'hard'
-    ]
 
-    individual_status = [
-        'mh'
+    RIVM_URL = "https://github.com/mzelst/covid-19/raw/master/data-rivm/tests/"
+
+    _required_params = [
+        "size",
+        "steps",
+        "actions",
+        "status",
+        "lockdown_policies",
+        "lockdown",
     ]
 
+    lockdown_policies = ["absent", "easy", "medium", "hard"]
+
+    individual_status = ["mh"]
+
     all_possible_features = [
-        'age_group__1',
-        'age_group__2',
-        'age_group__3',
-        'age_group__4',
-        'gender_f',
-        'gender_m',
-        'education_high',
-        'education_low',
-        'education_medium',
-        'unemployed_no',
-        'unemployed_yes',
-        'have_partner_no',
-        'have_partner_yes',
-        'depressed_no',
-        'depressed_yes',
-        'children_presence_no',
-        'children_presence_yes',
-        'housing_financial_difficulties_no',
-        'housing_financial_difficulties_yes',
-        'selfrated_health_average',
-        'selfrated_health_good',
-        'selfrated_health_poor',
-        'critical_job_no',
-        'critical_job_yes'
+        "age_group__1",
+        "age_group__2",
+        "age_group__3",
+        "age_group__4",
+        "gender_f",
+        "gender_m",
+        "education_high",
+        "education_low",
+        "education_medium",
+        "unemployed_no",
+        "unemployed_yes",
+        "have_partner_no",
+        "have_partner_yes",
+        "depressed_no",
+        "depressed_yes",
+        "children_presence_no",
+        "children_presence_yes",
+        "housing_financial_difficulties_no",
+        "housing_financial_difficulties_yes",
+        "selfrated_health_average",
+        "selfrated_health_good",
+        "selfrated_health_poor",
+        "critical_job_no",
+        "critical_job_yes",
     ]
 
     all_possible_actions = [
-        'work_from_home',
-        'maintain_physical_distance',
-        'stay_at_home',
-        'exercise',
-        'socialise',
-        'travel',
-        'seek_help',
-        'negative_coping',
-        'positive_coping',
-        'socialise_online'
+        "work_from_home",
+        "maintain_physical_distance",
+        "stay_at_home",
+        "exercise",
+        "socialise",
+        "travel",
+        "seek_help",
+        "negative_coping",
+        "positive_coping",
+        "socialise_online",
     ]
 
+    @staticmethod
+    def get_file_paths(url: str) -> List:
+        """
+        Extract file paths from url
+
+        Args:
+        url (str): website
+
+        Returns:
+        file_paths (list): list of '.csv.gz' file paths
+        """
+
+        response = requests.get(url)
+        # Will raise an HTTPError
+        # if the HTTP request was unsuccessful
+        response.raise_for_status()
+        data = response.json()
+        # extract '.csv.gz' file paths
+        file_paths = [item["path"] for item in data["payload"]["tree"]["items"]]
```

Comment:
Shouldn't this check for files with suffix `.csv.gz` to exclude other files explicitly? Not sure if that can happen, but it is suggested in the comment.

```suggestion
        file_paths = [item["path"] for item in data["payload"]["tree"]["items"] if item["path"].endswith(".csv.gz")]
```